cmd/deploy.go

```diff
@@ -51,25 +51,23 @@ func init() {
 	// deployment commands.
 	for _, remote := range config.Remotes {
 		cmd := &cobra.Command{
-			Use:    remote.Name + " [COMMAND]",
+			Use:    remote.Name + " [command]",
 			Hidden: true,
 			Short:  "Configure deployment to " + remote.Name,
 			Long: `Manage deployment on specified remote.
 
 Requires:
-1. an Inertia daemon running on your remote - use 'inertia [REMOTE] init'
-   to set one up.
-2. a deploy key to be registered for the daemon with your GitHub repository.
+1. an Inertia daemon running on your remote - use 'inertia [REMOTE] init' to get it running.
```

Comment:
`[REMOTE]` should now be lowercase `[remote]`

Reply:
🔥 6121b81